Events/Ch16.event

```diff
@@ -0,0 +1,180 @@
+UNIT Seth EphraimLord 0x00 Level(1, Ally, 0) [17,16] 0x00 0x00 0x0 0x00 [0x00] NoAI 
+UNIT Seth EphraimLord 0x00 Level(1, Ally, 0) [18,16] 0x00 0x00 0x0 0x00 [0x00] NoAI 
+UNIT Seth EphraimLord 0x00 Level(1, Ally, 0) [16,17] 0x00 0x00 0x0 0x00 [0x00] NoAI 
+UNIT Seth EphraimLord 0x00 Level(1, Ally, 0) [18,17] 0x00 0x00 0x0 0x00 [0x00] NoAI 
+UNIT Seth EphraimLord 0x00 Level(1, Ally, 0) [16,18] 0x00 0x00 0x0 0x00 [0x00] NoAI 
+UNIT Seth EphraimLord 0x00 Level(1, Ally, 0) [17,18] 0x00 0x00 0x0 0x00 [0x00] NoAI 
+UNIT Seth EphraimLord 0x00 Level(1, Ally, 0) [18,18] 0x00 0x00 0x0 0x00 [0x00] NoAI 
+UNIT Seth EphraimLord 0x00 Level(1, Ally, 0) [17,19] 0x00 0x00 0x0 0x00 [0x00] NoAI 
+UNIT
+
+/*Units:
+UNIT Eirika EirikaMasterLord 0x00 Level(6, Ally, 1) [17,17] 0x00 0x00 0x0 0x00 [IronSword,SteelSword,LightBrand,Vulnerary] NoAI 
+UNIT Seth Paladin 0x00 Level(6, Ally, 1) [16,16] 0x00 0x00 0x0 0x00 [Horseslayer,SteelLance,IronSword,SilverSword] NoAI 
+UNIT Gilliam General 0x00 Level(6, Ally, 1) [17,16] 0x00 0x00 0x0 0x00 [HeavySpear,IronLance,Javelin,Vulnerary] NoAI 
+UNIT Franz GreatKnight 0x00 Level(6, Ally, 1) [18,16] 0x00 0x00 0x0 0x00 [IronAxe,SilverLance,IronSword,HandAxe] NoAI 
+UNIT Moulder Bishop 0x00 Level(6, Ally, 1) [16,17] 0x00 0x00 0x0 0x00 [Shine,Mend,Restore,Sleep] NoAI 
+UNIT Garcia Hero 0x00 Level(6, Ally, 1) [18,17] 0x00 0x00 0x0 0x00 [SteelAxe,HandAxe,Vulnerary,PureWater] NoAI 
+UNIT Neimi Ranger_F 0x00 Level(6, Ally, 1) [16,18] 0x00 0x00 0x0 0x00 [KillerBow,IronSword,Longbow,Vulnerary] NoAI 
+UNIT Lute Sage_F 0x00 Level(6, Ally, 1) [17,18] 0x00 0x00 0x0 0x00 [Elfire,Fire,Lightning,Heal] NoAI 
+UNIT Joshua Swordmaster 0x00 Level(6, Ally, 1) [18,18] 0x00 0x00 0x0 0x00 [KillingEdge,LightBrand,IronSword,Elixir] NoAI 
+UNIT Knoll Druid 0x00 Level(6, Ally, 1) [17,19] 0x00 0x00 0x0 0x00 [Flux,Luna,Vulnerary,Vulnerary] NoAI 
+UNIT*/
+
+MariyaGroup:
+UNIT 0x15 WyvernKnight_F 0x00 Level(1, Ally, 0) [3,19] 0x00 0x00 0x1 MariyaREDA [ShortSpear,IronLance,PureWater] NoAI
+UNIT
+MariyaREDA:
+REDA [3,19] 0x0 0x0 0x0 0x0
+
+TurnBasedEvents:
+END_MAIN
+
+CharacterBasedEvents:
+CHAR 0xC MariyaTalk [0x15,0x1] 0x0
+CHAR 0xC MariyaTalk [0x1,0x15] 0x0
+END_MAIN
+
+LocationBasedEvents:
+Chest(MasterSeal,13,9)
+Chest(DelphiShield,5,2)
+Door(6,9)
+Door(7,9)
+Door(4,15)
+Door(5,15)
+END_MAIN
+
+MiscBasedEvents:
+CauseGameOverIfLordDies
+DefeatBoss(EndingScene)
+AREA 0xB MariyaLoad [0,0] [8,19]
+END_MAIN
+
+Dunno:
+//DO NOT TOUCH
+WORD $00
+
+Tutorial:
+//DO NOT TOUCH
+WORD $00
+
+TrapData:
+END_MAIN
+
+ALIGN 4
+
+BeginningScene:
+MUSS 0x4D
+//LOAD2 0x0 Units
+LOAD1 0x0 Units
+ENUN
+LOAD1 0x1 Enemies
+ENUN
+SVAL 0x2 0x34
+SVAL 0x3 BeginningTextCh16
+CALL 0x9EE310
+REMA
+GotoPrepScreen
+ENDA
+
+EndingScene:
+MUSS 0x32
+CAM1 0x1
+CUMO 0x1
+STAL 60
+CURE
+Text(VedgeWinsCh16)
+FADI 4
+SVAL 0x2 0x2C
+CHECK_ALIVE 0x15 //checks if mariya is alive
+BEQ 0x1400 0xC 0x0 //if not alive, send to label 1400
+SVAL 0x3 MariyaAliveCh16
```

Comment:
This works and is cool, but imo it's cleaner looking code-wise to the a `Text` macro, even if that does duplicate the underlying `CALL` code.